remove_odoo_enterprise/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Eska Yazılım ve Danışmanlık A.Ş (www.eskayazilim.com.tr)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Remove Odoo Enterprise",
+    "summary": "Remove enterprise modules and setting items",
+    "version": "14.0.1.0.0",
+    "category": "Maintenance",
+    "author": "Eska, Onestein, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/server-brand",
+    "license": "AGPL-3",
+    "depends": ["base"],
+    "external_dependencies": {"python": ["lxml"]},
```

Comment:
`lxml` it's already in standard Odoo: https://github.com/odoo/odoo/blob/14.0/requirements.txt#L22-L25
No need to add an external dependency in the manifest.

Reply:
removed!